examples/kubernetes/static-provisioning/volume/pv.yaml

```diff
@@ -19,4 +19,4 @@ spec:
     volumeAttributes:
       dnsName: "fs-078106df77a01f1f7.fsx.us-east-1.aws.internal"
       volumePath: "fsx"
-      volumeType: "volume"
+      type: "volume"
```

Comment:
Next revision I will fix these variables to be capitalized like their API equivalents.